pkg/promlib/instrumentation/instrumentation.go

```diff
@@ -18,8 +18,8 @@ const (
 	StatusOK    = "ok"
 	StatusError = "error"
 
-	EndpointCallResource = "callResource"
-	EndpointQueryData    = "queryData"
+	EndpointCallResource = string(backend.EndpointCallResource)
+	EndpointQueryData    = string(backend.EndpointQueryData)
```

Comment:
maybe just use the sdk types here rather than re-defining them?

Reply:
They might be used by other prometheus flavors such as amazon so didn't want to do it. I'll leave that as a follow up for @grafana/observability-metrics

Reply:
Currently there is no specific code for this in amazon-flavor but I think it is a good idea to have it that way. 
cc: @njvrzm please see this conversation ☝️